cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -818,18 +820,15 @@ <h2 class="h3 u-mb15">
             {% elif card.transfer_apr_median is not none
                 and card.transfer_apr_min != card.transfer_apr_max %}
                 <p>
-                    You’ll only know your exact balance transfer APR if you are
-                    approved for the card. The median balance transfer APR for
-                    this card is {{ apr(card.transfer_apr_median) }}.
+                    The median balance transfer APR for this card is
+                    {{ apr(card.transfer_apr_median) }}.
                 </p>
             {% endif %}
-            {# TODO: Explain what a grace period is #}
             <p>
-                The balance transfer APR is available for
-                {{ '%g' | format(card.median_length_of_balance_transfer_apr) }}
-                months after getting this card, on average. There is
-                {{ 'a' if card.balance_transfer_grace_period else 'no' }}
-                <a href="/ask-cfpb/do-i-pay-interest-on-new-purchases-after-i-get-a-zero-or-low-rate-balance-transfer-en-49/">balance transfer grace period</a>.
+                After
+                {{ '%g' | format(card.median_length_of_balance_transfer_apr | round) }}
+                months, any remaining unpaid balance on your account will be
+                subject to interest charges based on your purchase APR.
```

Comment:
*Clicks a [random card detail page ](http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/credit-union-of-colorado-a-federal-credit-union-platinum-preferred-rewards/)to check this formatting*

<img width="676" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/1060248/0813eeec-10b9-4780-a08a-953a81893cbf">

*Begins crying*

Reply:
At least you didn't find the card where the period is expressed as an exponential!